sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/appconfiguration/config/implementation/AppConfigurationReplicaClientsBuilder.java

```diff
@@ -169,7 +177,9 @@ List<AppConfigurationReplicaClient> buildClients(ConfigStore configStore) {
                 if (!credentialConfigured) {
                     // System Assigned Identity. Needs to be checked last as all of the above should
                     // have an Endpoint.
-                    LOGGER.debug("Connecting to {} using Azure System Assigned Identity or Azure User Assigned Identity.", endpoint);
+                    LOGGER.debug(
+                        "Connecting to {} using Azure System Assigned Identity or Azure User Assigned Identity.",
```

Comment:
Is it System/User Assigned **Managed** Identity?

Reply:
System, I think this comment/log is from the old 2.x version. Updated.